packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -143,3 +153,9 @@ export class DiagnosticFixPlugin extends Plugin {
     });
   }
 }
+
+function isInstallPackageCommand(
```

Comment:
We probably don't need this, because of currently CodeActionComand == InstallPackageAction :)

![image](https://user-images.githubusercontent.com/1502496/215564001-92c926cd-544b-4323-b4fc-6c2a4d7d3ebb.png)

but can keep it for future stability :) 

but can keep

Reply:
Also, I think

`isInstallPackageCommand(command: CodeActionCommand)`
instead 
`isInstallPackageCommand(fix: CodeFixAction)`

seems better :)